modules/test/playwright/helpers/HeadlessCommerceAdminOrderApiHelper.ts

```diff
@@ -119,19 +119,29 @@ export class HeadlessCommerceAdminOrderApiHelper {
 	}
 
 	async patchOrder(id: number, order: TOrder) {
-		const postOrder = await this.apiHelpers.patch(
+		await this.apiHelpers.patch(
 			`${this.apiHelpers.baseUrl}${this.basePath}orders/${id}?nestedFields=orderItems`,
 			order
 		);
 
+		const patchOrder = await this.apiHelpers.get(
+			`${this.apiHelpers.baseUrl}${this.basePath}orders/${id}`
+		);
```

Comment:
why do we need to do the get after the patch?

Reply:
because this patch doesn't return anything